src/backend/src/app.js

```diff
@@ -58,47 +52,27 @@ process.on('exit', (code) => {
  * @param {string} options.migrationsFolderPath
  * @param {string} options.sharedStoragePath Path to app-specific external file storage folder
  * @param {string} options.defaultConfigPath
- * @param {string} options.fallbackMapPath Path to app-specific external file storage folder
- *
  */
 export async function init({
   version,
   rootKey,
   migrationsFolderPath,
-  sharedStoragePath,
+  // sharedStoragePath,
```

Comment:
opted to just comment this out as opposed to remove since we'll need to use it for supporting custom maps in a follow-up that's coming soon